components/dashboard/conf/Caddyfile

```diff
@@ -0,0 +1,45 @@
+{
+	auto_https off
+	admin off
+}
+
+(compression) {
+	encode zstd gzip
+}
+
+(caching) {
+	header Cache-Control "public, max-age=604800, must-revalidate"
+}
+
+(default) {
+	header -Server
+}
+
+(discard_log) {
+	log {
+		output discard
+	}
+}
+
+:80 {
+	import caching
+	import compression
+	import default
+	import discard_log
+
+	redir /.well-known/security.txt https://www.gitpod.io/.well-known/security.txt permanent
+	redir /environment-variables /settings/ permanent
+	redir /login-success / permanent
```

Comment:
please remove this redirect.

Reply:
Without the redirect, I get a 404. Not sure if that's an error

Reply:
because of https://github.com/gitpod-io/gitpod/pull/3851/files#r610569110

Reply:
fixed